### PR TITLE
Alerts blog

### PR DESCRIPTION
**Pull request by sjmiller609**

### CODE REVIEW

File: src/content/blog/2024-05-30-introducing-alerts/index.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+slug: introducing-status-alerts
+title: 'PostgreSQL Alerts on Tembo Cloud'
```
Comment (samay-sharma) [inline src/content/blog/2024-05-30-introducing-alerts/index.mdx:3]: I would change the title to something better. What about something like:

* Introducing Database Alerts on Tembo Cloud
* Easier Postgres Management with Alerts on Tembo Cloud

**Review by samay-sharma — APPROVED** [review]
This blog reads good to me. I only have one suggestion around the title.

Can someone do a quick pass of:

* Grammar check
* See if there's an opportunity to add more links.

Other than that, I'm good with shipping this.

File: src/content/blog/2024-05-30-introducing-alerts/index.mdx (hunk 1 above)
Comment (bonesmoses) [reply src/content/blog/2024-05-30-introducing-alerts/index.mdx]: Second choice looks good to me. I went ahead and switched to that.

**Comment by bonesmoses** [comment]
I see that tying my author entry to this specific PR was probably a bad move, as it will complicate future contributions until it's merged unless we don't care about future merge conflicts. Doh!

**Pull request merged**